bin/rucio-admin

```diff
@@ -67,6 +68,17 @@ logger = logging.getLogger("user")
 tablefmt = 'psql'
 
 
+def unindent(text: object, indent: object = 0) -> object:
```

Comment:
The name `unindent` is misleading, since the actual unindentation starts from the second line. Maybe `unindent_without_first_line` or `unindent_after_first_line` or `unindent_from_second_line`?

Reply:
My original comment still stands: the need for this function is unclear. The `textwrap.dedent()` function from the standard library can be used directly by having the first line of the triple-quoted string be a backslash.

Reply:
Yes I agree. The example for the `dedent` function is clear: https://docs.python.org/3/library/textwrap.html#textwrap.dedent
The first line only has to be a backslash, and the next line should be indented as well. I'm also in favor of this change. :+1: